content/documents/search-index-tools.mdx

```diff
@@ -0,0 +1,73 @@
+---
+type:           article
+title:          Introducing Search Index Tools for Portal
+description:    A new set of tools for troubleshooting search index issues
+author:         edustin 
+publishDate:    2023-01-03
+updatedDate:    2023-01-03
+tags:           ["Search", "Portal"]
+---
+
+We're excited to unveil a long-requested enhancement for OrderCloud's premium search feature: a self-service tool for viewing and rebuilding your search indexes. By the end of this article, you should be ready to investigate and resolve basic search errors in your marketplace without the need for OrderCloud support.
+
+## A Background On Search Indexes
+
+The following section builds upon [previous articles](/knowledge-base?t=Search) for OrderCloud's premium search capabilities. Understanding these concepts will help you interpret your marketplace's search indexes.
+
+- Premium search endpoints serve cached Product and Order data from Elasticsearch to support greater search capabilities and faster performance.
+- Only one "active" search index per type can exist at any given time, and only an index with a status of `Ready` can be the active index for a marketplace. OrderCloud will point to a new index only after a successful rebuild.
+- Index errors typically arise when xp data types are not consistent between items within the same resource type. See the section on [xp schema errors](/knowledge-base/search-index-tools#understanding-xp-schema-errors) below for more detail.
+- Errors can exist on an active index. OrderCloud will fail an index build if the number of rejected items is considered too high or xp limits are exceeded.
+- A full index rebuild is a tool to "reset" your search index from the master OrderCloud record, and can oftentimes resolve data synchronization issues. This capability was previously limited to only the support team, but is now extended to marketplace administrators [with limitations](/knowledge-base/search-index-tools#rebuilding-your-index).
+
+## Troubleshooting With Search Index Tools
+
+Registered portal users now have visibility to search index details for their assigned marketplaces. After selecting a marketplace from the list, you will have the option to view either Product or Order search indexes from the tab selections. 
```

Comment:
above you say this is limited to Marketplace administrators, here you say registered portal users + assigned marketplaces. Let's be consistent/clear on who has access to these features.

Reply:
Any registered portal user can view search index details of marketplaces that they are a member of.  only administrators can trigger a rebuild.  I'll look at the language to make this clearer

Reply:
I'm not sure I agree with that choice - I feel like you should have to have a specific role to even see them.